packages/sdk/src/wait_for_remote_peer.ts

```diff
@@ -35,42 +32,48 @@ export async function waitForRemotePeer(
   timeoutMs?: number
 ): Promise<void> {
   protocols = protocols ?? getEnabledProtocols(waku);
+  const connections = waku.libp2p.getConnections();
```

Comment:
nit: should this be below the check for `isStarted()`?

Reply:
I don't see this being important as if the node is not started - then we will terminate, but if it is started the connections will be there

`isStarted` is not async so we don't risk to have old set of connections

and I added it on top to have variables assigned in one place for readability